src/lib.rs

```diff
@@ -10,16 +10,14 @@
 //! for writing applications but also to be a toolbox of networking primitives, so
 //! every layer is fully exposed and documented.
 //!
-//! When discussing networking stacks and layering, often the [OSI model][osi] is invoked.
+//! When discussing networking stacks and layering, often the [OSI model](osi) is invoked.
```

Comment:
Wait, what? How is this legal Markdown?

Reply:
Its not :(. I was playing around with some stuff and it got lumped in here.